### ISSUE TITLE
Improve preview/confirmation formatting

### ISSUE THREAD

**Issue by ngoldman** [ngoldman/gh-release#15]
Would be nice for the release preview to have better formatting. Currently it's just printing the object it'll be sending to the releases API:

```js
{ tag_name: 'v1.1.0',
  target_commitish: '394c232692d89ff3c904d4d3b046db4bc43c2aa0',
  name: 'v1.1.0',
  body: '* target node `0.12` and `iojs` on travis\n* user docker and cache `node_modules` on travis\n* add oauth2 token authentication method [#5](https://github.com/ngoldman/gh-release/issues/5)\n\n',
  owner: 'ngoldman',
  repo: 'gh-release',
  draft: false,
  prerelease: false }
```

I'm thinking it would be more readable like this:

```
repository: ngoldman/gh-release
release: v1.1.0
tag: v1.1.0
commit: 394c232692d89ff3c904d4d3b046db4bc43c2aa0
draft: false
prerelease: false
body:
* target node `0.12` and `iojs` on travis
* user docker and cache `node_modules` on travis
* add oauth2 token authentication method [#5](https://github.com/ngoldman/gh-release/issues/5)
```

**Comment by bcomnes** [comment]
:+1: 

Should we add colors?

**Comment by ngoldman** [comment]
We could if we want to get fancy. Could use https://www.npmjs.com/package/chalk

**Comment by ngoldman** [comment]
Resolved by @paulcpederson in #24. Thanks! :cool:
